test/framework/src/main/java/org/opensearch/test/FeatureFlagSetter.java

```diff
@@ -23,22 +23,42 @@
  */
 public class FeatureFlagSetter {
 
+    private static FeatureFlagSetter INSTANCE = new FeatureFlagSetter();
```

Comment:
shouldn't this be null if we are doing lazy initialisation?

Reply:
aha yes